pgjdbc/src/test/java/org/postgresql/test/jdbc2/DatabaseMetaDataTest.java

```diff
@@ -73,7 +73,6 @@ public void setUp() throws Exception {
     }
     TestUtil.createTable(con, "metadatatest",
         "id int4, name text, updated timestamptz, colour text, quest text");
-    TestUtil.createTable(con, "precision_test", "implicit_precision numeric");
```

Comment:
I think you might be basing this on an out of date branch. Rebase atop master as these changes should not be removing those tests.